pypeit/spectrographs/keck_lris.py

```diff
@@ -156,10 +165,15 @@ def compound_meta(self, headarr, meta_key):
         if meta_key == 'binning':
             binspatial, binspec = parse.parse_binning(headarr[0]['BINNING'])
             binning = parse.binning2string(binspec, binspatial)
+
+            # Hack for the not-stable commission version of the Mark4 detector
+            if binning == '0,0':
+                binning = '1,1'
```

Comment:
If we're including this hack, there should be a test for whether the data was taken before May 2021 and then proceed. Furthermore, `1,1` would have to be translated to `2,2`. And of course, any other binning mode (e.g. `0,1`) will have to be suitably translated. Instead of going that route, I'd suggest just looking at that date and raising a warning that the binning might be incorrect to the user.

Reply:
agreed. I am removing it.  At this stage, I'd like that data to 
crash.  The headers are simply no good.